doc/project/checklist.md

```diff
@@ -155,12 +155,12 @@ Review Design Change with SW: Review known "Won't Fix" bugs and "Errata".
 
 ## V1
 
-For a transition from V0 to V1, the following items are expected be completed.
+For a transition from V0 to V1, the following items are expected be completed. Prefix "SIM" is applicable for simulation-based DV approach only, while "FPV" is for FPV approach only.
 
 ### DV_PLAN_DRAFT_COMPLETED
 
 - DV Plan document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions,
-coverage objects.
+coverage objects (if applicable).
```

Comment:
Continue this on the previous line itself since it's part of the above sentence.

Reply:
Good catch, thanks Sri.